pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -1479,7 +1480,8 @@
         "3.9.0.0": "3.9.0",
         "4.0.0.0": "4.0.0",
         "4.0.10.0": "4.0.10",
-        "4.0.11.0": "4.0.11"
+        "4.0.11.0": "4.0.11",
+        "4.0.12.0": "4.3.0"
```

Comment:
The changes look correct. HOwever I would also expect to see a "4.0.12.1" to "4.3.1" mapping. However if not one is actually depending on the latest version it doesn't get used by anything.